### ISSUE TITLE
Prep project boards for Migration

### ISSUE THREAD

**Issue by Rabia2219** [Aveline-art/miniature-octo-sniffle#17]
### Overview
We need to prepare the Project Boards for the forced migration that GitHub is doing in August, so that it is already working well for the team by then.

### Action Items
Repeat first 3 steps for each card
- [ ]  copy this template to a comment below
   ````
   ### Text from project board card, [Add name of board here], [Add name of column here]
   ```
   [PASTE CARD CONTENT HERE]
   ```
   - [ ] Determine where this item goes on the wiki (page and section)
   - [ ] copy item to wiki
   - [ ] hide this comment when completed.
   ````
- [ ] copy the content of the card (by clicking the 3 dots and selecting Edit Note, and then CRTL+C) and paste it into your new comment where it says `[PASTE CARD CONTENT HERE]
- [ ] delete card from project board
- [ ] add project board URL(s) in resources below
- [ ] add the `project board migration` label to this issue
- [ ] add issue to project board
- [ ] move issue to Prioritized Backlog

### Resources/Instructions
Classic Project board URL(s):
- Open:
  - [Test Project](https://github.com/100Automations/GHA-add-issue-to-project/projects/1)

- Closed: None
New Project board URL(s): None

**Comment by ExperimentsInHonesty** [comment]
### Text from project board card, Test Project, To do
```
**Automation**
[Automatically move your cards](https://docs.github.com/articles/configuring-automation-for-project-boards/) to the right place based on the status and activity of your issues and pull requests.
```
- [x] Determine where this item goes on the wiki (page and section) - https://github.com/100Automations/GHA-add-issue-to-project/issues/18
- [ ] copy item to wiki
- [x] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
@Aveline-art 
We are in the process of migrating all the project boards to the new format. 

In this repo, you have an automation to add issues to the project board.  Because it was built with the classic project boards, we assume it will have to be restructured.  The notes on this issue are to document what is on the[ project board](https://github.com/100Automations/GHA-add-issue-to-project/projects/1), so that we have a record of it for the future, and to keep track of the changes that are made.  We will be closing the project board so that it does not get migrated automatically to the new board format until someone is ready to review this automation and update it.  You are being notified because you are the original developer.

There are two issues.  One open and one closed

- Open issue is in the In Progress column #13 
- Closed issue is in the Done column #12 

<details><summary>Picture of the board</summary>
<p>

![image](https://github.com/user-attachments/assets/8494075a-480d-4e5a-a6e3-2fb2d692ed6c)


</p>
</details>
